common/lib/xmodule/xmodule/js/src/capa/display.js

```diff
@@ -210,83 +210,73 @@
 
-            // Render 'x/y point(s)' if student has attempted question
-            if (status !== 'none' && (detail !== null && detail !== undefined) && (jQuery.type(detail) === 'string') &&
-                detail.indexOf('/') > 0) {
-                a = detail.split('/');
-                earned = parseFloat(a[0]);
-                possible = parseFloat(a[1]);
-                if (graded === 'True' && possible !== 0) {
+            if (curScore === undefined || totalScore === undefined) {
+                progress = '';
+            } else if (attemptsUsed === 0 || totalScore === 0) {
+                // Render 'x point(s) possible' if student has not yet attempted question
+                if (graded === 'True' && totalScore !== 0) {
+                    progressTemplate = ngettext(
+                        // Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10).;
+                        '%(num_points)s point possible (graded)', '%(num_points)s points possible (graded)',
+                        totalScore
+                    );
+                } else {
+                    progressTemplate = ngettext(
+                        // Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10).;
+                        '%(num_points)s point possible (ungraded)', '%(num_points)s points possible (ungraded)',
+                        totalScore
+                    );
+                }
+                progress = interpolate(progressTemplate, {num_points: totalScore}, true);
+            } else {
+                // Render 'x/y point(s)' if student has attempted question
+                if (graded === 'True' && totalScore !== 0) {
                     progressTemplate = ngettext(
                         // This comment needs to be on one line to be properly scraped for the translators.
                         // Translators: %(earned)s is the number of points earned. %(possible)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points);
                         '%(earned)s/%(possible)s point (graded)', '%(earned)s/%(possible)s points (graded)',
-                        possible
+                        totalScore
                     );
                 } else {
                     progressTemplate = ngettext(
                         // This comment needs to be on one line to be properly scraped for the translators.
-                        // Translators: %(earned)s is the number of points earned. %(possible)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points);
+                        // Translators: %(earned)s is the number of points earned. %(possible)s is the total number of
```

Comment:
I read through the i18n coding guidelines, and it does not say that translator comments in JS need to be on a single line. So this most likely will work, but it still makes me a tad nervous.